test/broadcasting.jl

```diff
@@ -0,0 +1,364 @@
+module TestBroadcasting
+
+using Test, DataFrames
+
+refdf = DataFrame(ones(3, 5))
```

Comment:
It would be more robust to use random values: when they are all equal, the code could be completely broken (e.g. recycle the first value) and the tests wouldn't catch it.

Reply:
I have made explicit non-random tests now (that should be relatively safe to recycling while allowing "eye" checking of the correctness of the result)